data/remnant/remnant missions.txt

```diff
@@ -3445,3 +3445,274 @@ event "remnant: pantica void sprites"
 event "remnant: cognizance timer"
 	system "Nenia"
 		fleet "Ember Waste Node" 6000
+
+
+
+mission "Remnant: Expanded Horizons Astral 1"
+	name "Preparation for expedition"
+	description "To prepare for an expedition to Sagittarius A*, go to <destination> with 6 outfit space available."
+	passengers 1
+	minor
+	to offer
+		has "Remnant: Expanded Horizons Quarg 3: done"
+		# random < 25
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	on offer
+		conversation
+			`"In the starport cafeteria you come across Dawn sitting at a table looking worried. She smiles when she sees you, and waves you to the seat opposite her. "Back from gallivanting around the galaxy with more tales to share?" she signs.`
+			choice
+				`	"Yes, as a matter of fact..."`
+				`	"No, just hunting Korath..."`
+				`	"No, just testing some stuff..."`
+				`	"You look worried. Is something happening?"`
+					goto worried
+			`	She listens for a minute or two as you tell her about what you have been up to, but it is obvious that she has something on her mind.`
+			choice
+				`	"You look worried. Is everything alright?"`
+					goto worried
+				`	"You look distracted. We can talk later."`
+					defer
+			label worried
+			`	She makes a gesture of appreciation. "Yes, I'm worried, <first>. I keep feeling like there's something I'm missing." She pauses, then continues. "I should back up a step. While I do not have a permanent position yet, I am currently serving with an astrophysics team. We have been studying a variety of things, and I have been tasked with reviewing our bulk astrophysical data. It is fairly routine,  but I keep getting this prickly feeling that there's a problem that I am missing. Nothing obvious, of course. Our neural networks would pick up something obvious or repetitious really quickly. But the feeling just will not go away."`
```

Comment:
> "Yes, I'm worried, <first>. I keep feeling like there's something I'm missing." She pauses, then continues. "I should back up a step. While I do not have a permanent position yet, I am currently serving with an astrophysics team.
"She pauses, then continues," feels like it's written for the sake of being written. More description would be good.

Reply:
Hmmm... I'll change up that a bit. I keep going over it, not sure what to do. But we'll see what the change does.